devicetypes/smartthings/osram-lightify-gardenspot-mini-rgb.src/osram-lightify-gardenspot-mini-rgb.groovy

```diff
@@ -21,9 +21,6 @@ metadata {
         attribute "colorName", "string"
 
         command "setAdjustedColor"

```

Comment:
the setAdjustedColor method actively set the level value to empty. we kinda replicate that behavior in the setColor method by actively choosing just the hue and saturation value